src/client/components/notifications.vue

```diff
@@ -85,6 +86,20 @@ export default defineComponent({
 	mounted() {
 		this.connection = os.stream.useSharedConnection('main');
 		this.connection.on('notification', this.onNotification);
+
+		// queueに対してのみ既読処理を行う
+		this.connection.on('readAllNotifications', () => {
+			this.queue = this.queue.map(markNotificationRead);
```

Comment:
markNotificationReadの引数が変わったときにバグるから
```suggestion
			this.queue = this.queue.map(x => markNotificationRead(x));
```
って書いた方が良さそう